AspNetCoreReactEnvironments/ClientApp/package.json

```diff
@@ -18,23 +19,25 @@
   "devDependencies": {
     "ajv": "^6.9.1",
     "cross-env": "^5.2.0",
-    "typescript": "^3.7.5",
     "eslint": "^6.8.0",
     "eslint-config-react-app": "^5.2.0",
     "eslint-plugin-flowtype": "^4.6.0",
     "eslint-plugin-import": "^2.20.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.18.3"
+    "eslint-plugin-react": "^7.18.3",
+    "typescript": "^3.7.5"
   },
   "eslintConfig": {
     "extends": "react-app"
   },
   "scripts": {
-    "start": "rimraf ./build && react-scripts start",
-    "build": "react-scripts build",
+    "start": "cross-env CLIENT_ENV=development craco start",
+    "build": "cross-env CLIENT_ENV=production craco build",
+    "staging": "cross-env CLIENT_ENV=staging craco build",
```

Comment:
change `react-script` to `craco`
pass environment name via `CLIENT_ENV` parameter